i3pystatus/taskwarrior.py

```diff
@@ -80,7 +80,8 @@ def run(self):
 
         if self.next_task is not None:
             format_values['next'] = self.next_task['description']
-            format_values['project'] = self.next_task['project']
+            format_values['project'] = (self.next_task['project'] if 'project' in self.next_task
+                                        else '')
```

Comment:
It might be nicer to do:

```python
self.next_task.get('project', '')
```

https://docs.python.org/3.1/library/stdtypes.html#dict.get